scripts/release_scripts/generate_release_info.py

```diff
@@ -67,7 +67,10 @@ def get_current_version_tag(repo):
     Returns:
         github.Tag.Tag: The most recent version tag.
     """
-    return repo.get_tags()[0]
+    if 'hotfix' in common.get_current_branch_name():
+        return repo.get_tags()[1]
```

Comment:
It might be possible to have more than one hotfix for a given release. Will this still work in that case? (Please add tests for this case too.)

Reply:
Yes, it will since in case of multiple hotfixes as well, the first tag will be of current release and the second of previous release.

(For example, for release-2.8.7, the tag will be v2.8.7, now even if we have two hotfixes, the first tag is v2.8.7 and second v2.8.6, so this function will return v2.8.6 irrespective of the number of hotfix)

Added a test as well.

Reply:
As discussed, please add a detailed comment explaining conceptually what's going on here and why [1] and [0] are the correct indexes to look at respectively.

Reply:
Added.